incubator/concourse/README.md

```diff
@@ -0,0 +1,243 @@
+| `concourse.genericOauthAuthUrlParam` | Parameters (comma separated) to pass to the authentication server AuthURL | `nil` |
+| `concourse.genericOauthScope` | Optional scope required to authorize user | `nil` |
+| `concourse.genericOauthTokenUrl` | Generic OAuth provider TokenURL endpoint | `nil` |
+| `web.nameOverride` | Override the Concourse Web components name | `web` |
+| `web.replicas` | Number of Concourse Web replicas | `1` |
+| `web.resources` | Concourse Web resource requests and limits | `{requests: {cpu: "100m", memory: "128Mi"}}` |
+| `web.service.type` | Concourse Web service type | `NodePort` |
+| `web.ingress.enabled` | Enable Concourse Web Ingress | `false` |
+| `web.ingress.annotations` | Concourse Web Ingress annotations | `{}` |
+| `web.ingress.hosts` | Concourse Web Ingress Hostnames | `[]` |
+| `web.ingress.tls` | Concourse Web Ingress TLS configuration | `[]` |
+| `worker.nameOverride` | Override the Concourse Worker components name| `worker` |
+| `worker.replicas` | Number of Concourse Worker replicas | `2` |
+| `worker.minAvailable` | Minimun number of workers available after an eviction | `1` |
+| `worker.resources` | Concourse Worker resource requests and limits | `{requests: {cpu: "100m", memory: "512Mi"}}` |
+| `persistence.enabled` | Enable Concourse persistence using Persistent Volume Claims | `true` |
+| `persistence.worker.class` | Concourse Worker Persistent Volume Storage Class | `generic` |
+| `persistence.worker.accessMode` | Concourse Worker Persistent Volume Access Mode | `ReadWriteOnce` |
+| `persistence.worker.size` | Concourse Worker Persistent Volume Storage Size | `10Gi` |
+| `postgresql.postgresUser` | PostgreSQL User to create | `concourse` |
+| `postgresql.postgresPassword` | PostgreSQL Password for the new user | `concourse` |
+| `postgresql.postgresDatabase` | PostgreSQL Database to create | `concourse` |
+| `postgresql.persistence.enabled` | Enable PostgreSQL persistence using Persistent Volume Claims | `true` |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml incubator/concourse
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+### SSH Keys
+
+To run Concourse securely you'll need [3 private keys](https://concourse.ci/binaries.html#generating-keys). For your convenience, this chart provides some default keys, but it is recommended that you generate your own keys by running:
+
+```console
+$ mkdir -p concourse-keys
+$ ssh-keygen -t rsa -f concourse-keys/host_key -N '' -C concourse
+$ ssh-keygen -t rsa -f concourse-keys/session_signing_key -N ' -C concourse
+$ ssh-keygen -t rsa -f concourse-keys/worker_key -N '' -C concourse
+```
+
+And update the `values.yaml` file with the generated keys:
```

Comment:
@viglesiasce Wouldn't it be easier if I just update the README? So instead of `And update the values.yaml file with the generated keys` will be something like `And provide the generated keys by using helm install --set privateKey= cat id_rsa  --set publicKey= cat id_rsa.pub`?